visualizer/parseTree.js

```diff
@@ -10,6 +12,9 @@
                root.domUtil);
   }
 })(this, function(ohm, ohmEditor, CheckedEmitter, document, cmUtil, d3, domUtil) {
+  // Fake getComputedStyle() for node (required for unit tests)
+  var getComputedStyle = getComputedStyle || function(elt) { return {}; };
```

Comment:
This is the purpose of the `initModule` stuff above. You should add a new argument named `getComputedStyle` after `domUtil`, and then pass in the stub version from the unit test itself.